### ISSUE TITLE
ClientSocketHandler::check_origin() fails for http://127.0.0.1:8888

### ISSUE THREAD

**Issue by agwells** [catalyst/websockets-dashboard#76]
python 2.7
tornado 4.1

I downloaded this project and tried to run it using the instructions in the README.md and the default config-dist.ini values:

    [general]
    host=127.0.0.1
    port=8888

But when I visited http://127.0.0.1:8888 in my browser, all I saw was the "Connection lost, please refresh browser" message. The browser was sending a request to http://127.0.0.1:8888/chatsocket, which was returning a 403 Forbidden.

Google suggested that Tornado giving a 403 forbidden, is due to the "same origin" check, in the ClientSocketHandler::check_origin() method, in server/server.py. This method just checks whether 'http://'+host == origin

I added some logging to it to find out the values of host and origin. And it turned out host == "127.0.0.1" and origin = "http://127.0.0.1:8888". So it was failing because the "host" value lacked the port number.

I'm a python newb so I don't have a patch to fix this yet.

**Comment by srynot4sale** [comment]
Yeah this is a dumb thing we did. On the "prod" dashboard it sits behind nginx so is therefore behind port 80 so 'http://'+host == origin will work.

For dev you can just return True from that function.

We had a patch for this but it was causing issues from memory.
